packages/slider/styles.css

```diff
@@ -0,0 +1,113 @@
+:root {
+  --reach-slider: 1;
+  --highlight-color: #1159a6;
+}
+
+[data-reach-slider] {
+  padding: 20px;
+}
+
+[data-reach-slider-orientation="horizontal"] {
+  width: 250px;
+  height: 4px;
+  padding-bottom: 30px;
+}
+
+[data-reach-slider-orientation="vertical"] {
+  width: 4px;
+  height: 250px;
+  padding-right: 30px;
+}
+
+[data-reach-slider-track] {
+  border-radius: 2px;
+  background: #f2f2f2;
+  width: inherit;
+  height: inherit;
+}
+
+[data-reach-slider-handle] {
+  width: 12px;
+  height: 12px;
+  border: 2px solid var(--highlight-color);
+  background: #fff;
+  border-radius: 10px;
+  z-index: 1;
+  transform-origin: center;
+  transition: transform 200ms ease-out;
+}
+
+[data-reach-slider-handle-orientation="horizontal"] {
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+[data-reach-slider-handle-orientation="horizontal"]:focus {
+  transform: translateY(-50%);
+}
+
+[data-reach-slider-handle-orientation="vertical"] {
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+[data-reach-slider-handle-orientation="vertical"]:focus {
+  transform: translateX(-50%);
+}
+
+[data-reach-slider-track-highlight] {
+  border-radius: inherit;
+  background: var(--highlight-color);
+}
+
+[data-reach-slider-track-highlight-orientation="horizontal"] {
+  height: 100%;
+}
+
+[data-reach-slider-track-highlight-orientation="vertical"] {
+  width: 100%;
+}
+
+[data-reach-slider-marker] {
+  width: 8px;
+  height: 8px;
+  background: inherit;
+  border: 1px solid #fff;
+  border-radius: 4px;
+  transform-origin: center;
+  transition: background 200ms ease-out;
```

Comment:
This color transition seems like it's a little too opinionated for this level. It's nice, but I think it belongs in a higher level design system.

Reply:
Removed